SU-SWS/decanter
# ADAPT-5863 | Update to Source Sans 3 and Source Serif 4; minor updates

# READY FOR REVIEW

# Summary
- Update font stack and import to Source Sans 3 and Source Serif 4, with the old versions as fallback (in the mixin and classes)
- Remove ie11 from browerslistrc
- linter fix

# Needed By (Date)
- Soonish

# Urgency
- 3

# Steps to Test

1. Look at code
2. The preview is not really tied to this PR unfortunately, but you could pull this branch down locally, and do `npm run dev`, `npm run start`, and inspect the text like I did here. It should tell you the rendered fonts are Source Sans 3
![Screenshot 2023-07-25 at 2 57 12 PM](https://github.com/SU-SWS/decanter/assets/42749717/27092ee6-8f0d-40c8-9405-172340df6674)


# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- ADAPT-5795, https://stanfordits.atlassian.net/browse/ADAPT-5863
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
